src/components/InputTodo/index.js

```diff
@@ -0,0 +1,29 @@
+import { IoMdAddCircle } from 'react-icons/io';
+import PropTypes from 'prop-types';
+import styles from './InputTodo.module.css';
+
+const InputTodo = ({ handleTasksChange }) => {
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const title = event.target.children[0].value;
+    handleTasksChange(title);
+    event.target.reset();
+  };
```

Comment:
- Kindly remove the async keyword in the handleSubmit, because there are no data to wait for, it's just waiting till the form submit and React handles this behind the scene by using event handlers.